website/static/css/style.css

```diff
@@ -743,3 +743,9 @@ div.ball-scale-blue>div {
 #componentQuickActions a {
     color: #333;
 }
+
+/* counter */
+.counter-comment {
+  margin-top: -20px;
```

Comment:
This will need a bit of adjustment to avoid text overlapping
<img width="256" alt="screen shot 2017-11-10 at 5 18 21 pm" src="https://user-images.githubusercontent.com/1322421/32681132-ee87a8f0-c63b-11e7-867d-6a02b6efd52f.png">